python/nano/notebooks/pytorch/tutorial/pytorch_lightninglite.ipynb

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
the tutorial should be about how a PyTorch program with custom train loop can be easily accelerated by nano, not about lightning-lite

Reply:
changed it